### PR TITLE
WIP Benoit/images opensuse

### PR DESCRIPTION
**Pull request by vaudelbenoit**

### CODE REVIEW

**Comment by danielobyrne** [comment]
Actually just noticed that there's a line that is incorrect. Should be BYOS not BOYS

File: source/images/images-by-catalyst.rst [hunk 1]
```diff
@@ -43,10 +47,18 @@ as the default user name you should use to log in to each one of them
...
+
+    <a href="https://en.opensuse.org/openSUSE:License" target="_blank">opensuse.org</a>
+
 .. |ubuntu_link| raw:: html
 
     <a href="https://ubuntu.com/legal/intellectual-property-policy" target="_blank">ubuntu.com</a>
 
+.. note::
+
+  BOYS: Bring Your Own Subscription
```
Comment (puck) [inline source/images/images-by-catalyst.rst:60]: Typo, should be BYOS.

File: source/images/images-by-catalyst.rst [hunk 2]
```diff
@@ -24,6 +24,10 @@ as the default user name you should use to log in to each one of them
 +---------------------+-------------------+-----------+-------------------+
 | Fedora Atomic Linux | fedora-atomic-*   | fedora    | |fedora_link|     |
 +---------------------+-------------------+-----------+-------------------+
+| OpenSuse Leap Linux | opensuse-leap-*   | opensuse  | |opensuse_link|   |
++---------------------+-------------------+-----------+-------------------+
+| Suse SLES Linux     | suse-sles-*       | sles      | BYOS model        |
```
Comment (puck) [inline source/images/images-by-catalyst.rst:29]: Suse should be SUSE

File: source/images/images-by-catalyst.rst [hunk 3]
```diff
@@ -24,6 +24,10 @@ as the default user name you should use to log in to each one of them
 +---------------------+-------------------+-----------+-------------------+
 | Fedora Atomic Linux | fedora-atomic-*   | fedora    | |fedora_link|     |
 +---------------------+-------------------+-----------+-------------------+
+| OpenSuse Leap Linux | opensuse-leap-*   | opensuse  | |opensuse_link|   |
```
Comment (puck) [inline source/images/images-by-catalyst.rst:27]: OpenSuse should be openSUSE.

**Review by puck — COMMENTED** [review]
Hey, a few things to fix.

File: source/images/images-by-catalyst.rst (hunk 1 above)
Comment (puck) [reply source/images/images-by-catalyst.rst:60]: Heh, Daniel spotted it as well.

**Pull request merged**